src/util/scopeToKeyword.ts

```diff
@@ -0,0 +1,40 @@
+import { ScopeType } from "../typings/Types";
+
+export type ScopeTypeToKeyword = { [scope in ScopeType]: string };
+
+const scopeToKeyword: ScopeTypeToKeyword = {
```

Comment:
Heyo — I thought that it might be nice to have a layer of indirection between the scheme files and the scopeTypes instead of a direct mapping. I'll think about some unlocks that came to mind when writing this an get back to you.

Reply:
Got it — thoughts on converting `ScopeType` into an [`enum`](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) type? I want to be able to iterate through it and match against the query file.